index.html

```diff
@@ -144,27 +144,14 @@ <h2 class="section-title" id="add-task-header">Add New Task</h2>
         <section class="task-list">
           <h2 class="section-title">My Task List</h2>
 
-          <table class="table-box">
-            <tr>
-              <th class="table-box-color-tag"></th>
-              <th>Task</th>
-              <th>Time</th>
-              <th></th>
-            </tr>
-            <tr class="table-box-row">
-              <td class="table-box-color-tag-job"></td>
-              <td class="table-box-task">Maria Anders</td>
-              <td class="table-box-time">10:10 AM</td>
-              <td class="table-box-check-box">
-                <input
-                  class="table-box-check"
-                  type="checkbox"
-                  id="checkbox-done"
-                  name="checkbox-done"
-                />
-              </td>
-            </tr>
-          </table>
+          <ul class="table-box">
+            <ul class="table-header-box">
```

Comment:
`ul` cannot have `ul` as a child.

Reply:
Oh,you are right,I changed my parent `ul` to `div`